src/components/EditorHeader/EditorHeader.vue

```diff
@@ -0,0 +1,218 @@
+      />
+      <SbMobileViwer
+        v-if="returnViewFormat === 'mobile'"
+        v-bind="{
+          languages,
+          users,
+          actions,
+        }"
+        @changes="$_handleListenersOrchestrator"
+      />
+      <div
+        class="sb-editor-header__content"
+        :class="{
+          'sb-editor-header__content--mobile': returnViewFormat === 'mobile',
+        }"
+      >
+        <SbHeaderItem v-if="hasSaveButton" with-separator>
+          <button
+            class="sb-editor-header__save-button"
+            @click="handleSaveChanges"
+          >
+            Save
+          </button>
+        </SbHeaderItem>
+
+        <SbHeaderItem
+          v-if="spaceStatus"
+          with-separator
+          class="sb-editor-header__status"
+        >
+          <SbIcon
+            :name="returnNameOfIconStatus"
+            size="large"
+            :color="returnColorOfStatus"
+          />
+        </SbHeaderItem>
+
+        <SbHeaderItem>
+          <SbButton
+            size="small"
+            label="Publish"
+            type="primary"
+            @click="handlePublishSpace"
+          />
+        </SbHeaderItem>
+
+        <SbHeaderItem v-if="options">
+          <SbMenu>
+            <SbMenuButton has-icon-only />
+            <SbMenuList placement="bottom-end">
+              <SbMenuItem
+                v-for="option in options"
+                :key="option.id"
+                :type="option.type"
+                @click="handleSelectOptions(option.name)"
+              >
+                {{ option.name }}
+              </SbMenuItem>
+            </SbMenuList>
+          </SbMenu>
+        </SbHeaderItem>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import SbDesktopAndTabletViwer from './components/DesktopAndTabletViwer'
+import SbMobileViwer from './components/MobileViwer'
+import SbHeaderTitle from './components/HeaderTitle'
+
+import SbHeaderItem from './components/HeaderItem'
+import SbIcon from '../Icon'
+
+import SbButton from '../Button'
+
+import { SbMenu, SbMenuButton, SbMenuList, SbMenuItem } from '../Menu'
+
+import { sharedProps } from './lib'
+
+export default {
+  name: 'SbEditorHeader',
+
+  components: {
+    SbDesktopAndTabletViwer,
+    SbHeaderTitle,
+    SbMobileViwer,
+    SbButton,
+    SbHeaderItem,
+    SbIcon,
+    SbMenu,
+    SbMenuButton,
+    SbMenuList,
+    SbMenuItem,
+  },
+
+  props: {
+    ...sharedProps,
+  },
+
+  data: () => ({
+    window: {
+      width: 0,
+    },
+  }),
+
+  computed: {
+    returnNameOfIconStatus() {
+      return this.spaceStatus === 'published'
+        ? 'published-circle'
+        : 'status-circle'
+    },
+
+    returnColorOfStatus() {
+      return this.spaceStatus === 'unpublished'
+        ? 'light'
+        : this.spaceStatus === 'published'
+        ? 'primary'
+        : this.spaceStatus === 'published-and-changes'
+        ? 'light-gray'
+        : 'light'
+    },
+
+    returnViewFormat() {
```

Comment:
Maybe we need to create something more elaborate, or use some plugin to define breakpoints. But let's leave it at that for now.